RevenueCatUI/PaywallView.swift

```diff
@@ -106,9 +138,14 @@ public struct PaywallView: View {
     ) {
         self._introEligibility = .init(wrappedValue: introEligibility ?? .default())
```

Comment:
can probably assert that either offering or offeringSelection or none is expected.

Reply:
The current logic allows to have any combination of the above. The `offering` will take precedence over the `offerSelection`.